app/server.go

```diff
@@ -146,7 +146,7 @@ type Server struct {
 	telemetryService *telemetry.TelemetryService
 
 	remoteClusterService     *remotecluster.Service
-	sharedChannelSyncService *sharedchannel.Service
+	sharedChannelSyncService SharedChannelServiceIFace
```

Comment:
What's the reason for turning this into an interface?

Reply:
it was done in order to be able to mock the service. Maybe i am missing something but returning a concrete type of `*sharedchannel.Service` will not allow us to do so.

Reply:
Within the shared channel service package an IFace for the service can be created.  You can auto-generate a mock for that Iface there, or just manually create a simple one.  There is already a mock for server in `services/remotecluster/mock_test.go` that can return your service mock.  That would keep with the idiom of returning concrete types while accepting interfaces, and reduce proliferation of interfaces between packages.

That said, we're under time crunch so I'll accept as is. Thanks.